predrill.md

```diff
@@ -3,47 +3,47 @@
 
 {{BOM}}
 
-The box will be made of one Top, one bottom, 2 long sides and 2 short sides. In this step, we will drill holes in the Top and long sides plexiglass plates.
+The box will be made of one top, one bottom, 2 long sides and 2 short sides. In this step, we will drill holes in the top, short and long sides plexiglass plates.
+
+
+
+>i **Note** 
+>i
+>i All constructional drawings have a higher resolution if you open them in a new tab.
```

Comment:
I do not think so, is it GItbuilding feature I am not aware of ?

Reply:
if you zoom in your browser, it shows that the resolution has not changed, only the pic is smaller